apps/data-scanner/lib/models/sessions/Sessions.mjs

```diff
@@ -35,31 +38,86 @@ export class Sessions {
      * @returns {Promise<void>}
      */
     async init() {
-        const sessions = await new Session().readAll();
+        const sessions = await new Session().readAll({ valid: true });
+
+        this.#pool['full'] = [];
+        this.#pool['avaliable'] = [];
 
-        for (const session of sessions) {
-            const client = new Client(session, this.#config); //todo: extend Client class with the sessionId prop
+        for (let session of sessions) {
+            const client = await new Client(session, this.#config);
             await client.init();
-            this.#pool.push(client); // todo: replace array with object { sessionId: client }
-            // [{sessionId, api }, {sessionId, api }, {sessionId, api }]
-            // { 
-            //     1234: {sessionId:1234, api },
-            //     3456: {sessionId:3456, api },
-            // }
+            if (session.is_full) this.#pool['full'].push({ sessionId: session.id, client });
```

Comment:
this.#pool.filled.set(session.id, { sessionId: session.id, client });
else
this.#pool.available.set(session.id, { sessionId: session.id, client });